### PR TITLE
propperly broadcast changes to settings and environment variables

### PR DESCRIPTION
**Pull request by mwrock**
This fixes https://github.com/chef/chef-dk/issues/534 by broadcasting changes to the environment variables. Note that this is dependent on https://github.com/chef/omnibus/pull/558.

/cc @chef/omnibus-maintainers

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    hv.cpus = 2
+  end
+
+  config.vm.provider :virtualbox do |vb|
+    vb.gui = true
+    vb.memory = 2048
+    vb.cpus = 2
+    vb.network "forwarded_port", guest: 5985, host: 55985
+  end
+end
```
Comment (schisamo) [inline Vagrantfile]: Why are we introducing an `Vagrantfile` here? All other platforms (including Windows) build with the included `.kitchen.yml`.

Comment (mwrock) [inline Vagrantfile]: ugh. this was added by mistake. I'll yank it. Sorry about that. I use this for personal windows omnibus development.

**Comment by mwrock** [comment]
removed my Vagrantfile

File: Vagrantfile (hunk 1 above)
Comment (schisamo) [inline Vagrantfile]: @mwrock If the `.kitchen.yml` is lacking or not working we should chat about how we can fix that WRT to Windows.

**Comment by jaym** [comment]
:+1:

**Comment by schisamo** [comment]
:+1:

**Comment by mwrock** [comment]
Will merge as soon as I validate another build. Hoping it finishes before the holidays.

**Comment by danielsdeleo** [comment]
:tada: nice work

**Comment by ksubrama** [comment]
Hey, this breaks with
```
 [Packager::MSI] I | $ light.exe -nologo -ext WixUIExtension -ext 'WixUtilExtension' -cultures:en-us -loc "C:\Users\jenkins\AppData\Local\Temp\chefdk20151020-2360-48s0bk\localization-en-us.wxl" project-files.wixobj source.wixobj -out "C:\omnibus-ruby\pkg\chefdk-0.9.0+20151020085008-1-x86.msi"
                                    I | C:\Users\jenkins\AppData\Local\Temp\chefdk20151020-2360-48s0bk\source.wxs(33) : error LGHT0094 : Unresolved reference to symbol 'CustomAction:WixBroadcastSettingChange' in section 'Product:*'.

                                    I | C:\Users\jenkins\AppData\Local\Temp\chefdk20151020-2360-48s0bk\source.wxs(34) : error LGHT0094 : Unresolved reference to symbol 'CustomAction:WixBroadcastEnvironmentChange' in section 'Product:*'.

The following shell command exited with status 94:

    $ light.exe -nologo -ext WixUIExtension -ext 'WixUtilExtension' -cultures:en-us -loc "C:\Users\jenkins\AppData\Local\Temp\chefdk20151020-2360-48s0bk\localization-en-us.wxl" project-files.wixobj source.wixobj -out "C:\omnibus-ruby\pkg\chefdk-0.9.0+20151020085008-1-x86.msi"

Output:

    C:\Users\jenkins\AppData\Local\Temp\chefdk20151020-2360-48s0bk\source.wxs(33) : error LGHT0094 : Unresolved reference to symbol 'CustomAction:WixBroadcastSettingChange' in section 'Product:*'.

C:\Users\jenkins\AppData\Local\Temp\chefdk20151020-2360-48s0bk\source.wxs(34) : error LGHT0094 : Unresolved reference to symbol 'CustomAction:WixBroadcastEnvironmentChange' in section 'Product:*'.

Error:

    (nothing)
```

**Comment by mwrock** [comment]
Just ran another build on a fresh builder with success. I strongly think this is due to an old version of the wix toolset on the build machines. They should get updated with the latest wix cookbook which installs 3.10

**Pull request merged**